tests/integration/test_issue_1933.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+import subprocess
+
+import pytest
+
+from pex.common import is_exe
+from pex.testing import run_pex_command
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+@pytest.mark.skipif(
+    not any(
+        is_exe(os.path.join(entry, "docker"))
```

Comment:
`shutil.which` would be easier to read. It was introduced in cp3.3, though it looks like we don't need to run this test with 2.7

Reply:
It may be, but actually using means a conditional import / it would be cleanest to provide a `pex.compatibility.which` shim, but that's for prod and this is only done in tests. At the end of the day, since Pex supports 2.7 the right thing to do is accept that and write in 2.7. I do this daily and flip the Python stdlib webdite api docs drop-down back to 2.7 to make sure I'm using APIs that I can actually use safely.

Reply:
I'll stop beating this drum :)